src/librepgp/stream-packet.cpp

```diff
@@ -1616,6 +1616,9 @@ stream_parse_signature_body(pgp_packet_body_t *pkt, pgp_signature_t *sig)
     }
 
     /* signature MPIs */
+    if (sig->palg == PGP_PKA_EDDSA && sig->version < PGP_V4) {
+        RNP_LOG("Warning! v3 EdDSA signature.");
+    }
```

Comment:
The same as previous - no need to change code while it is enough to add a comment.